tensorflow/cc/tools/freeze_saved_model.cc

```diff
@@ -171,6 +172,28 @@ void ConvertReadVariableOpToIdentity(const NodeDef& node,
   identity_node->add_input(node.input(0));
 }
 
+// Returns the name of the VarHandleOp input of a ReadVariableOp. Considers that
+// the VarHandleOp is not necessarily an immediate input to ReadVariableOp, the
+// handle name can be forwarded by identity ops.
```

Comment:
How about changing the comment to this:

Returns the name of the VarHandleOp that provides the input to a chain of Identity op. This can help locating the handle name indirectly input to ReadVariableOp through a chain of Identity op.